README.md

```diff
@@ -1,2 +1,36 @@
-# rfcs
-RFCs for Strapi future changes
+# Strapi RFCs
+
+Some important changes in Strapi require some thoughts to be put into the design phase before starting workin on a PR.
+
+The RFC (Request For Comments) process will help us create consensus among the core team and include as much feedback as possible from the community.
```

Comment:
The RFC (Request For Comments) process will help us create consensus among the core team and include as much feedback as possible from the community, for these upcoming changes.